dev-aws/kafka-shared-msk/energy-budget-plan/energy-budget-plan.tf

```diff
@@ -152,3 +152,15 @@ module "budget_plan_assessment" {
   consume_groups   = ["energy-budget-plan.assessment-v1"]
   cert_common_name = "energy-budget-plan/assessment"
 }
+
```

Comment:
Seems the file needs formatting with terraform format, or you can use pre-commit : https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/CONTRIBUTING.md#linting

Reply:
The lint job is failing: https://github.com/utilitywarehouse/kafka-cluster-config/actions/runs/9545118711/job/26305219807?pr=372